### PR TITLE
fix: a11y for placeholder

### PR DESCRIPTION
**Pull request by vincentaudebert**
## Type

- Enhancement

#### What is expected?

A11y for placeholder component

#### The following changes where made:

(Describe what you did)

1. Add progress cursor

2. Add media query for soft/no animation

3. Add comment into storybook to remind about accessibility for Placeholder container.

### CODE REVIEW

File: src/components/Placeholder/index.tsx [hunk 1]
```diff
@@ -94,7 +100,11 @@ Placeholder.propTypes = {
 }
 
 Placeholder.defaultProps = {
+  col: undefined,
```
Comment (chambo-e) [inline src/components/Placeholder/index.tsx:103]: Why ? 🤔

File: src/components/Placeholder/index.tsx [hunk 2]
```diff
@@ -69,17 +81,11 @@ const Placeholder: React.VoidFunctionComponent<PlaceholderProps> = ({
   const Component = variants[variant]
 
   return (
-    <Box
-      position="relative"
-      display="block"
-      width="100%"
-      overflow="hidden"
-      {...props}
```
Comment (chambo-e) [inline src/components/Placeholder/index.tsx:77]: You sure ? There is a lot of Placeholder that still use xstyled props :)

File: src/components/Placeholder/index.tsx (hunk 1 above)
Comment (vincentaudebert) [reply src/components/Placeholder/index.tsx:103]: To satisfy linter on commit :/

Comment (vincentaudebert) [reply src/components/Placeholder/index.tsx:103]: ie. 

95:3   error    propType "col" is not required, but has no corresponding defaultProps declaration     react/require-default-props
  96:3   error    propType "height" is not required, but has no corresponding defaultProps declaration  react/require-default-props
  97:3   error    propType "length" is not required, but has no corresponding defaultProps declaration  react/require-default-props
  99:3   error    propType "width" is not required, but has no corresponding defaultProps declaration   react/require-default-props

Comment (vincentaudebert) [reply src/components/Placeholder/index.tsx:103]: Oh wait. I need to remove defaultProps completely 😂

**Review by chambo-e — APPROVED** [review]

**Review by matthprost — APPROVED** [review]

**Pull request merged**